app/controllers/groups/create.js

```diff
@@ -0,0 +1,48 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default class extends Controller {
+
+  @service errorHandler;
+
+  groupName ='';
+
+  groupEvents = [];
```

Comment:
Why storing it here? You already have group model, Use `group.events`

That's why it is not switching correctly